### ISSUE TITLE
confused with result

### ISSUE THREAD

**Issue by SonOfCoding** [microsoft/singleshotpose#120]
I have run your code for training driller (use your pretrained model).  For the last epoch, the result is
 
2019-10-20 16:00:14    Mean corner error is 13.450474
2019-10-20 16:00:14    Acc using 5 px 2D Projection = 44.70%
2019-10-20 16:00:14    Acc using 0.0262 vx 3D Transformation = 33.50%
2019-10-20 16:00:14    Acc using 5 cm 5 degree metric = 23.79%
2019-10-20 16:00:14    Translation error: 0.059544, angle error: 9.375706
2019-10-20 16:00:14 save training stats to backup/driller/costs.npz

are these results correct?
the 2D reprojection error is 44.70% , however, the table 1 in your paper shows 2D reprojection error is 79.41? how do you get 79.41? did I do something wrong?  please give me a help, thank you

**Comment by btekin** [comment]
Hi, the code has been refactored and updated to work with Python 3 and Pytorch 0.4, however some config files were not updated. This is what I see when I run the following:

`python valid.py --datacfg cfg/driller.data --modelcfg cfg/yolo-pose.cfg --weightfile backup/driller/model_backup.weights`

Results of driller
Acc using 5 px 2D Projection = 78.00%
Acc using 10% threshold - 0.026147216404619446 vx 3D Transformation = 64.82%
Acc using 5 cm 5 degree metric = 68.38%
Mean 2D pixel error is 4.293773, Mean vertex error is 0.028071, mean corner error is 6.832153
Translation error: 0.027343 m, angle error: 4.473990 degree, pixel error:  4.293773 pix

There is only a slight difference with the initial results in the paper, as the models had to be retrained after paper submission, due to slight variability of the training performance over multiple training sessions.
